test/check-application

```diff
@@ -72,6 +72,23 @@ class TestApplication(testlib.MachineCase):
         # check exited alpine not in running list
         b.wait_not_in_text("#containers-containers", "alpine:latest")
 
+        # delete running container busybox using force delete
+        b.click('#containers-containers tbody tr:contains("busybox:latest") + tr button.btn-delete')
+        self.confirm_modal("btn-danger")
+        self.confirm_modal("btn-danger")
+        b.wait_not_present(".modal-dialog div")
```

Comment:
Right now you have a race condition between the two `confirm_modal()` calls. The function itself doesn't wait for the dialog to close, and you also don't call this in between the two dialogs. So the second call would still see the first dialog before it closes.

I suggest to move this `wait_not_present` into `confirm_modal()`, to avoid forgetting about it.